languages/csharp/sensitivity/reflection/reflection.tmt

```diff
@@ -0,0 +1,31 @@
+~main body start~
+~helper import sensitivity/reflection/helpers/ReflectionHelper~
+
+~imports start~
+using System.Reflection;
+using HelperReflection;
+~imports end~
+
+~template simple_reflection_positive start~
+try {
+    var type = typeof(ReflectionHelper);
+    ReflectionHelper rh = type.GetConstructors()[0].Invoke(~[VAR_String@1]~);
```

Comment:
1) use `string` instead of `String`
1) `Invoke` method accepts `object?[]?`, try `Invoke([~[VAR_string@1]~])` 
1) `Invoke` method returns `object`, try explicit cast `(ReflectionHelper)`